docs/guide/release.md

```diff
@@ -0,0 +1,132 @@
+# Releasing a new version
+
+Releasing a new version of the SIX Web Components is a task performed by members of the Core Team,
+and upon discussion with other team members. The following document outlines the different steps
+required.
+
+It is recommended that you read through this document in its entirety before you start with the
+process!
+
+## Preparation
+
+Before you start with the release, there are a few steps that need to be undertaken.
+
+### Merging PRs
+
+Before you start, you need to make sure that all PRs that need to go in this release are merged!
+Check the list of open PRs [here](https://github.com/six-group/six-webcomponents/pulls), and chase
+down the authors if needed.
+
+### Test changes locally
+
+To avoid issues once the process has been started, it is good practice to test the changes locally:
+
+1. Checkout `main`
+2. Navigate to the root of the project
+3. Run `npm ci`. This will ensure that the required packages are correctly installed
```

Comment:
You can run npm install in local, npm ci is only intended for the ci environment :)